qa/acceptance/spec/shared_examples/cli/logstash-plugin/install.rb

```diff
@@ -44,32 +44,64 @@
             command = logstash.run_command_in_path("bin/logstash-plugin install #{gem_path_on_vagrant}")
             expect(command).to install_successfully
             expect(logstash).to have_installed?("logstash-filter-dns")
+            expect(logstash).not_to be_running
+            logstash.start_service
+            Stud.try(40.times, RSpec::Expectations::ExpectationNotMetError) do
+              expect(logstash).to be_running
+            end
+            logstash.stop_service
```

Comment:
here, we are adding a secondary validation that Logstash _can_ start up after performing the primary task successfully.

Since we are doing it repeatedly, I think we should break this out into a helper method.